### PR TITLE
fix(integrity): add support for loading multiple EVM x509 certs

### PR DESCRIPTION
**Pull request by stefanberger**
Add support for loading EVM x509 certs from a directory that the user can
specify with the EVMKEYSDIR variable in the evm config file. By default
the additional certs are loaded from /etc/keys/evm.

Support for multiple EVM keys allows the usage in a system of files with
signed metadata from multiple parties.

Signed-off-by: Stefan Berger <stefanb@linux.ibm.com>
Reviewed-by: Roberto Sassu <roberto.sassu@huawei.com>

This pull request changes...

## Checklist
- [X] I have tested it locally
- [X] I have reviewed and updated any documentation if relevant
- [ ] I am providing new code and test(s) for it

Fixes #

### CODE REVIEW

**Review by johannbg — APPROVED** [review]
LGTM

File: modules.d/98integrity/evm-enable.sh [hunk 1]
```diff
@@ -96,13 +94,24 @@ load_evm_x509() {
...
-    if ! EVMX509ID=$(evmctl import "${EVMX509PATH}" "${evm_pubid}"); then
-        info "integrity: failed to load the EVM X509 cert ${EVMX509PATH}"
-        return 1
+    if [ -z "${EVMKEYSDIR}" ]; then
+        EVMKEYSDIR="/etc/keys/evm"
     fi
+    # load the default EVM public key onto the EVM keyring along
+    # with all the other ones in $EVMKEYSDIR
+    PUBKEY_LIST=$(ls "${NEWROOT}"${EVMKEYSDIR}/*)
+    for PUBKEY in ${EVMX509PATH} ${PUBKEY_LIST}; do
```
Comment (haraldh) [inline modules.d/98integrity/evm-enable.sh:103]:
```suggestion
    for PUBKEY in ${EVMX509PATH} "${NEWROOT}"${EVMKEYSDIR}/*; do
```

**Review by haraldh — CHANGES_REQUESTED** [review]

File: modules.d/98integrity/evm-enable.sh (hunk 1 above)
Comment (stefanberger) [reply modules.d/98integrity/evm-enable.sh]: Done.

**Comment by stefanberger** [comment]
@haraldh Did I address your change request sufficiently?

**Review by haraldh — APPROVED** [review]

**Pull request merged**